src/sub-blocks/PriceCard/PriceCard.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+import Check from '@gravity-ui/icons/Check';
+
+import {Button, CardBase, ContentList, Link as LinkBlock} from '../../components';
+import {PriceCardProps} from '../../models';
+import {block} from '../../utils';
+
+import './PriceCard.scss';
+
+const b = block('price-card');
+
+const PriceCard = (props: PriceCardProps) => {
```

Comment:
maybe rewrite like this :
```
({border, ...}) => (<CardBase>....</CardBase>)
```

Reply:
I can see that other cards have this kinda syntax too. Let's keep it.